pkg/testing/builder/clusterpodplacementconfig.go

```diff
@@ -34,3 +35,22 @@ func (p *ClusterPodPlacementConfigBuilder) WithLogVerbosity(logVerbosity common.
 func (p *ClusterPodPlacementConfigBuilder) Build() *v1beta1.ClusterPodPlacementConfig {
 	return p.ClusterPodPlacementConfig
 }
+
+func (p *ClusterPodPlacementConfigBuilder) WithNodeAffinityScoring(enabled bool) *ClusterPodPlacementConfigBuilder {
+	if p.Spec.Plugins.NodeAffinityScoring == nil {
+		p.Spec.Plugins.NodeAffinityScoring = &plugins.NodeAffinityScoring{}
+	}
+	p.Spec.Plugins.NodeAffinityScoring.BasePlugin.Enabled = enabled
+	return p
+}
+
+func (p *ClusterPodPlacementConfigBuilder) WithNodeAffinityScoringTerm(architecture string, weight int32) *ClusterPodPlacementConfigBuilder {
+	if p.Spec.Plugins.NodeAffinityScoring == nil {
+		p.Spec.Plugins.NodeAffinityScoring = &plugins.NodeAffinityScoring{}
+	}
+	p.Spec.Plugins.NodeAffinityScoring.Platforms = append(p.Spec.Plugins.NodeAffinityScoring.Platforms, plugins.NodeAffinityScoringPlatformTerm{
+		Architecture: architecture,
+		Weight:       weight,
+	})
+	return p
```

Comment:
```suggestion
	}
	platform := plugins.NodeAffinityScoringPlatformTerm{
	  Weight: weight,
	}
	if architecture != "" {
	  platform.Architecture = architecture
	}
	p.Spec.Plugins.NodeAffinityScoring.Platforms = append(p.Spec.Plugins.NodeAffinityScoring.Platforms, platform)
	return p
```

To accomodate the test case in https://github.com/openshift/multiarch-tuning-operator/pull/369/files#diff-efb81a87029bf9712fa52e99dd6323a7f3ec4e09132bbe1fbcc480a3a0a96cf8R632-R636